bpo-38692: Skip test_posix.test_pidfd_open() on EPERM (#17290)
python/cpython: Misc/NEWS.d/next/Tests/2019-11-20-15-42-06.bpo-38692.aqAvyF.rst

> Skip test_posix.test_pidfd_open() test if ``os.pidfd_open()`` fails with a

Reviewer: ```suggestion
Skip the test_posix.test_pidfd_open() test if ``os.pidfd_open()`` fails with a
```

> :exc:`PermissionError`. It can happen in a Linux sandbox using a syscall

Reviewer: ```suggestion
:exc:`PermissionError`. This situation can happen in a Linux sandbox using a syscall
```

> whitelist which doesn't allow ``pidfd_open()`` syscall yet.

Reviewer: ```suggestion
whitelist which doesn't allow the ``pidfd_open()`` syscall yet.
```
